task/04-arrays-tasks.js

```diff
@@ -436,7 +479,7 @@ function sortCitiesArray(arr) {
  *           [0,0,0,0,1]]   
  */
 function getIdentityMatrix(n) {
-   throw new Error('Not implemented');
+   return Array(n).fill(Array(n).fill(0)).map((x,i)=>((x.map((y,j)=>(i==j)? y=1:y=0))));
```

Comment:
if you assign 1 to y it won't change the value in the array. Because y is value not a reference. The solution works, but assigning 1 and 0 to y is redundant. Also remove redundant parentheses.